onebusaway-transit-data-federation/src/main/java/org/onebusaway/transit_data_federation/impl/realtime/gtfs_realtime/GtfsRealtimeSource.java

```diff
@@ -366,7 +375,21 @@ private FeedMessage readFeedFromUrl(URL url) throws IOException {
       }
     }
   }
-
+/**
+ * Set the headers to the urlConnection if any
+ * @param urlConnection
+ * @return, the urlConnection with the headers set
+ */
+  private void setHeadersToUrlConnection(URLConnection urlConnection)
+  {
+	  if(_headersMap!= null)
```

Comment:
Actually, braces should be shifted too to match style.  I was looking at this on mobile and didn't see that at first.